content/en/docs/storage-providers/seal-workers.md

```diff
@@ -14,7 +14,7 @@ While the Lotus Miner runs each of the sealing phases itself by default, you can
 
 ## Resource allocation in Lotus Workers
 
-Each **Lotus Worker** can run multiple tasks, depending on your hardware resources. Each slot is called a _window_. The final number is determined by the number of available cores and the requirements of the sealing phase allocated to it. That means that a single worker on a 8-core CPU with a single GPU will run at most:
+Each **Lotus Worker** can run multiple tasks, depending on your hardware resources. Each slot is called a _window_. The final number is determined by the number of available cores and the requirements of the sealing phase allocated to it, including:
```

Comment:
Thought I should also mention this change is related to [issue #1021 in filecoin-docs](https://github.com/filecoin-project/filecoin-docs/issues/1021). I didn't change the bulleted list at the time myself because I didn't know enough about `lotus-worker` to do so, and while at this point I could probably make that list accurately, I'm not sure it's particularly useful to include here, because 
1. it's heavily dependent on RAM. Some minimal amount (e.g. 256GB) could be specified, but that might have the unfortunate effect of suggesting that one can/should run a worker on a machine with that little memory. I suppose some larger amount could be arbitrarily chosen, but then why stick with an 8-core CPU? That seems unrepresentative of what the majority of storage providers appear to be running anyway. 
2. it can be extrapolated from [Hardware Requirements](https://github.com/filecoin-project/lotus-docs/blob/main/content/en/docs/storage-providers/hardware-requirements.md). That makes me realize a link to that page might be helpful here. Perhaps I should update this PR to include that?